src/app/Main/Header/Header.component.tsx

```diff
@@ -28,7 +29,7 @@ const mainHeaderStyle = (open: boolean): SerializedStyles => css`
     justify-content: space-between;
     box-sizing: border-box;
     box-shadow: ${open ? "0 8px 6px -6px grey" : "none"};
-    z-index: 10;
+    z-index: 1;
```

Comment:
still global please remove.
The header should not be responsible of it's z position. It is the responsibility of the parent component (some layout?) to arrange the components. See also comment in QueryPageLayout how to do that.